src/cheri/hypervisor-integration.adoc

```diff
@@ -51,8 +51,6 @@ and <<misa>>.{CRE}=1, CHERI can be enabled by VS-mode and
 VU-mode. When <<henvcfg_cheri>>.{CRE}=0, CHERI is disabled in VS-mode and
 VU-mode as described in xref:section_cheri_disable[xrefstyle=short].
 
-The reset value is 0.
```

Comment:
I think this does actually need to stay - we want to ensure CRE is disabled on reset.

Reply:
ok - reworded